### PR TITLE
updated translations (Esperanto and Polish)

### PR DESCRIPTION
**Pull request by verdulo**
The text 'international' on the main page is bit uncool, English is not international language…

### CODE REVIEW

**Comment by joto** [comment]
"International" is not about the language. It is about the data that is in this taginfo instance and that's worldwide data. Please stick to actual translations in a translation pull request. If you think some wording on the page needs to change, open a separate issue and we can discuss this. It would have to be done in all translations anyway.

**Comment by verdulo** [comment]
So the better word would be 'Worldwide', also there should be somewhere information to maintainers of other taginfo instances to change that string, so they would show the name of supported country.

File: web/i18n/eo.yml [hunk 1]
```diff
@@ -213,7 +213,7 @@ pages:
         packed: Kompaktigita
         unpacked: Nekompaktigita
         description: Priskribo
-        note: Kelkaj indeksoj ne troviĝas en la datumbazo elŝutebla tie ĉi. La 'kompaktiga' versio ne inkludas tiujn ĉi indeksojn, la 'nekompaktigita' versio inkludas indeksojn, kiujn vi probable bezonos krei post elŝuto.
+        note: Kelkaj indeksoj ne troviĝas en la datumbazo elŝutebla tie ĉi. La ‘kompaktiga’ versio ne inkludas tiujn ĉi indeksojn, la ‘nekompaktigita’ versio inkludas indeksojn, kiujn vi probable bezonos krei post elŝuto.
```
Comment (joto) [inline web/i18n/eo.yml]: Is there a reason you are changing the quotes here and elsewhere?

File: web/i18n/eo.yml [hunk 2]
```diff
@@ -328,11 +328,11 @@ pages:
     relation:
         name: Rilat-tipo
         description_from_wiki: Priskribo de tiu ĉi rilat-tipo de la vikio (se disponebla en elektita lingvo, alie en la angla).
-        no_description_in_wiki: Neniu priskribo en la angla pri tiu ĉi rilat-tipo en la vikio. (Vidu ankaŭ la langeton 'Vikio'.)
+        no_description_in_wiki: Neniu priskribo en la angla pri tiu ĉi rilat-tipo en la vikio. (Vidu ankaŭ la langeton ‘Vikio’.)
         button_disabled: Butono malaktiva, ĉar estas tro multe da objektoj kun tiu ĉi etikedo.
         overview:
-            tab: Sumiĝo
-            title: Sumiĝo
+            tab: Sumigo
```
Comment (joto) [inline web/i18n/eo.yml]: Why is the accent gone here? Is that correct?

File: web/i18n/pl.yml [hunk 3]
```diff
@@ -419,7 +517,7 @@ reports:
     wiki_pages_about_non_existing_keys:
         name: Strony wiki na temat nieistniejących kluczy
         intro: |
-            Ta tabela pokazuje klucze, które posiadają swoje strony na wiki, ale nie istnieją nawet pojedyncze obiekty w bazie z ich użyciem. Nie musi być to błąd, gdyż ktoś mógł stworzyć dokumentację przed wdrożeniem nowego schematu tagowania. Ale może być to zwykła literówka albo inny błąd.
+            Ta tabela pokazuje klucze, które posiadają swoje strony na wiki, ale nie istnieją nawet pojedyncze obiekty w bazie z ich użyciem. Nie musi być to błąd, gdyż ktoś mógł stworzyć dokumentację przed wdrożeniem nowego schematu tagowania. Ale może być to zwykła literówka albo inny błąd.
```
Comment (joto) [inline web/i18n/pl.yml]: No spaces at the end please.

File: web/i18n/eo.yml [hunk 4]
```diff
@@ -37,7 +37,7 @@ taginfo:
     wiki: Vikio
     report: Raporto
     reports: Raportoj
-    international: Internacia
+    international: Interŝtata
```
Comment (joto) [inline web/i18n/eo.yml]: Why this change?

Comment (verdulo) [inline web/i18n/eo.yml]: nacio - nation, ŝtato - country

File: web/i18n/eo.yml (hunk 1 above)
Comment (verdulo) [inline web/i18n/eo.yml]: I started paying attention to proper punctuation, use apostrophes (') as apostrophes and quotes (‘’) as quotes, I know, that they look almost the same.

File: web/i18n/eo.yml (hunk 2 above)
Comment (verdulo) [inline web/i18n/eo.yml]: Yes.
